### DISCUSSION TITLE
Odin Question: Does Odin Start perform Steam validation?

### DISCUSSION THREAD

**Discussion by wolffstarr** [mbround18/valheim-docker#163] [Q&A] [answered]
I'm looking to implement the server lag fix mod - we routinely have 5-7 players and the server (and connection) can easily handle the upstream. This requires modifying the assembly_valheim.dll file, which Steam validate will overwrite. I can work around this with permissions, but I'd rather not change ownership on one file if there's no reason to, as that causes update headaches down the road.

**Comment by mbround18** [comment, marked as answer]
Odin does not pass the validate flag, mod until your hearts content <3

**Reply by wolffstarr** [reply]
Excellent, thanks for confirming for me!

**Comment by wolffstarr** [comment]
I haven't tried it yet - just heard about it yesterday and located the DLL
files last night - didn't want to deploy until I knew whether I would need
to block it from being replaced or not.
